tests/test_game.py

```diff
@@ -13,6 +13,65 @@ def test_from_arrays():
     assert len(game.players[1].strategies) == 2
 
 
+def test_empty_array_to_arrays():
```

Comment:
We still need tests for correctly converting to usual types (`float`, `Rational`), but also a test to ensure raising exception on a bad type (can use the `dict` example from above).